mcda/utils/utils_for_plotting.py

```diff
@@ -8,6 +8,10 @@
 import plotly.io as pio
 from PIL import Image
 
+import mcda.utils.utils_for_main as utils_for_main
+
+DEFAULT_OUTPUT_DIRECTORY_PATH = './output_files'  # root directory of ProMCDA
```

Comment:
similar to input.
Also, you have declared variable stating `dir` but the folder seems to have `files` in it. Probably refactor to have either `dir` or `files` in both the places?
`./output_files' is not root directory of ProMCDA.

Reply:
The files inside the directory are created by the run of ProMCDA. I think that here it's correct to declare a dir.